### DISCUSSION TITLE
Xray absorption correction

### DISCUSSION THREAD

**Discussion by olliepye** [mboquien/cigale#36] [General]
Hi All,

What is the impact of not using absorption corrected Xray fluxes? Would it simply be that the Xray flux would be lower than if absorption corrected and hence, that lower xray flux would propagate through to galaxy properties (ie. under-estimated Xray properties)? 

If it is essential to include absorption corrected fluxes, what would be the best way to implement this. Most X-ray catalogues provide a hardness ratio. Is there a simple method of applying the hardness ratio to determine the absorption factor? I have seen papers that use software such as XSPEC/PIMMS to estimate the absorption, but I do not have a background in Xray astronomy so any advice is greatly appreciated.

Thanks,
Ollie

**Comment by Guang91** [comment]
We have a suggestion for this (footnote 2 in Yang et al. 2022):

"For users without intrinsic X-ray fluxes, it is feasible to estimate the absorption corrections on their own (see, e.g., §3.1 of Mountrichas et al. 2021b). To perform this task, users can first use BEHR (Park et al. 2006) to estimate the hardness ratios (HRs) based on hard and soft-band counts, which are
often available in X-ray catalogs. They can then input these HR values to PIMMS (Mukai 1993) for the estimations of column density NH and intrinsic fluxes. An alternative approach is to directly adopt the hard-band fluxes without absorption corrections, because hard X-ray photons are penetrating and only modestly affected by absorption in general. For our case, the median correction for hard-band fluxes is only ≈ 5%."
(PIMMS has a handy web version https://cxc.harvard.edu/toolkit/pimms.jsp)

So, I'd try using hard-band fluxes directly and see the fitting results.

**Reply by olliepye** [reply]
Thank you!
